components/Tile/index.module.scss

```diff
@@ -0,0 +1,75 @@
+@import "styles/index";
+
+$shadow: 0 2px 22px 0 rgba(0, 0, 0, 0.15);
+
+.tile {
+  position: relative;
+  overflow: hidden;
+  display: inline-flex;
+  flex-direction: column;
+  background-color: white;
```

Comment:
Overall, this looks great. A minor question I have is, do we want to tie in this white to the __colors '$color-white' variable?